consumer/consumer.go

```diff
@@ -27,16 +27,17 @@ const (
 	// commitModulo is used to define the batch size of offsets based on the current offset being processed
 	commitModulo = 10
 
-	OperationTypeCreated   = "created"
-	OperationTypeUpdated   = "updated"
-	OperationTypeDeleted   = "deleted"
-	OperationTypeMigration = "migration"
+	OperationTypeReportResource = "ReportResource"
+	OperationTypeDeleteResource = "DeleteResource"
```

Comment:
I'm wondering if we can use the Kessel Go SDK to get a handle on these names 🤔 That way we can tightly couple them to the API/SDK.

Same thing for version, but I'm less certain we'll be able to do that without adding support for that explicitly.